build.gradle

```diff
@@ -323,6 +368,8 @@ clean {
     it.dependsOn 'cleanRoot'
 }
 
+
```

Comment:
remove this.

Reply:
Removed in 9258f11e90ded8d2193db5c2e4b1ee4edaa4ee2c